modules/formatter/tests/src/FormatterSpec.scala

```diff
@@ -583,6 +599,27 @@ final class FormatterSpec extends munit.FunSuite {
                       |        B
                       |    ]
                       |    errors: [E1, E2]
+                      |    errors: [E3]
+                      |}
+                      |
+                      |operation SomeOp {
+                      |    errors: [
+                      |        E1
```

Comment:
just noticed that we're preferring the lack of commas :<

I guess I'll live with it somehow

Reply:
nothing is fixed: we can discuss it

also it's worth exploring a way to configure the formatter a bit

Reply:
I'm fine with anything really, given the commas are now optional it might be objectively better. We can think about configuration later on ;)